lib/movable_ink/aws/metadata.rb

```diff
@@ -3,59 +3,55 @@
 module MovableInk
   class AWS
     module Metadata
-      def http
-        @http ||= begin
+      def http(timeout_seconds: 1)
+        @http = begin
```

Comment:
Are you sure you no longer want to memoize this?

Reply:
Yes, because we can't set a longer timeout if it's memoized and I don't want to mutate the memoized version. I'm open to other options though.

Reply:
Ah, that's a good point.  It's going to create a bunch of `Net::HTTP` instances